docs/source/concepts/architecture/tests/test-sectors.rst

```diff
@@ -0,0 +1,57 @@
+Test Sectors for GeoIPS
+=======================
+
+This document describes the naming conventions and implementation details for test sector plugins in GeoIPS.
```

Comment:
I'd prefer not referring to the document inside the document. How about something more like this:

"To improve the efficiency of integration tests for GeoIPS we make use of a number of small, pre-determined test sectors. These sectors were chosen to provide a variety of scene components including coast and land, edge of geostationary disk, day/night/terminator, etc. These sectors can be found in the ___________ directory."

Reply:
Fixed